### ISSUE TITLE
[bz4788267] Child errors are not communicated clearly to parents

### ISSUE THREAD

**Issue by add0n** [YahooArchive/mojito#25]
The default error() function the composite plugin uses to handle child errors looks like this:

``` javascript
        error: function(err) {
            Y.log("Error executing child mojit at '" + this.id + "':", 'error', NAME);
            Y.log(err.message, 'error', NAME);
            Y.log(err.stack, 'error', NAME);
            // Pass back some empty data so we don't fail the composite
            this.done('');
        }
```

In order to allow the parent mojit to continue running, it passes back an empty string into ac.done(), but we should have a way to communicate child errors

**Comment by caridy** [comment]
This is still an issue. Let's bump this up. /cc @rwaldura @jenny

**Comment by caridy** [comment]
will be available in mojito@0.5.5 scheduled for next week.
